align/compiler/match_graph.py

```diff
@@ -188,17 +188,17 @@ def _group_cap_const(self, G1, name):
 
         """
         if self._if_const(name):
-            const_list = self.hier_graph_dict[name]["const"]["constraints"]
+            const_list = self.hier_graph_dict[name]["constraints"]
```

Comment:
["const"] was always a dictionary with a single key ("constraints"). Eliminated this pseudo-layer in the heirarchy.